resources/views/cruds/datagrids/filters/datagrid-filter.blade.php

```diff
@@ -20,9 +20,9 @@
     @if ($activeFilters > 0)
         @if (empty($bookmark) && isset($entityType))
         @can('create', \App\Models\Bookmark::class)
-            <div class="inline-block cursor-pointer btn2 btn-sm btn-primary break-keep" data-toggle="dialog" data-target="datagrid-bookmark" data-url="{{ route('filters.modal_form', [$campaign, $entityType, 'm' => $mode]) }}">
+            <div class="inline-block cursor-pointer btn2 btn-sm btn-primary break-keep" data-toggle="dialog" data-target="datagrid-filters" data-url="{{ route('filters.modal_form', [$campaign, $entityType, 'm' => $mode]) }}">
```

Comment:
The div needs a role="button" and the aria-label on it to be accessible to screen readers, otherwise they won't know it's a button.